src/pages/FlameScans/main.ts

```diff
@@ -99,13 +100,16 @@ export const FlameScans: pageInterface = {
         con.error('404');
         return;
       }
-      if (j.$('h1.entry-title').length && j.$('.epcheck').length) {
+      if (
+        j.$('.mantine-Title-root').length &&
+        j.$('[class*="ChapterCard_chapterWrapper"]').length
```

Comment:
Is not part of the initial html source code, meaning it will not be triggered if the js loads it faster than malsycn can.

![image](https://github.com/user-attachments/assets/72d5bef0-cb89-4e5d-8a32-8ba36e8a18bd)